test/unit/has_slug_tests.rb

```diff
@@ -38,12 +38,36 @@ def update_column(name, value)
 
       value  = Factory.string
       record = subject.new.tap{ |r| r.send("#{@source_attribute}=", value) }
-      assert_instance_of Proc, subject.ardb_has_slug_config[:source_proc]
-      assert_equal value, record.instance_eval(&subject.ardb_has_slug_config[:source_proc])
+      source_proc = subject.ardb_has_slug_config[:source_proc]
+      assert_instance_of Proc, source_proc
+      assert_equal value, record.instance_eval(&source_proc)
```

Comment:
@jcredding I'd prefer to see that, by default, the source proc returns the source attribute.  Having "value" in the assertion is kinda an indirection of that.  Maybe this instead?

```ruby
record = subject.new.tap{ |r| r.send("#{@source_attribute}=", Factory.string) }
# ...
exp = record.send(@source_attribute)
assert_equal exp, record.instance_eval(&source_proc)
```

This is just more explicit about showing that the source proc returns what you set up above - the source attr of the record.